src/data_vis.ts

```diff
@@ -108,6 +108,8 @@ class DataVisModel {
     ko.observable(metricsAndOptions.uiVisible.includes("uncertainty"));
   private showSidebar = ko.observable(true);
 
+  private paper1Embargo = ko.observable(!this.isPrivate() && (Date.now() < Date.UTC(2021, 0, 28, 23, 30)));
```

Comment:
should that be a 2 in the second position?

Reply:
The second parameter to `UTC(...)`?- no, it's month, zero-indexed, unlike day or hour... :shrug: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC